src/seed/index.js

```
@@ -0,0 +1,36 @@
+      console.log('Database dropped successfully!');
+    }
+
+    await checkUserExistence();
+    await SeedUsers.createUsers();
+    await SeedCategories.createCategories();
+    await SeedSubjects.createSubjects();
+
+    console.log('Database seeded successfully!');
+    process.exit(0);
+  } catch (error) {
+    logger.error('Seeding error:', error); 
```

ESLint

[error] Extra semicolon.

(semi)